xml/deployment_installing_nodes.xml

```diff
@@ -387,6 +386,122 @@
   </procedure>
  </sect1>
 
+ <sect1 xml:id="sec.deploy.nodes.admin_install_cli">
+  <title>Installing the &Admin_Node; with Command Line Interface</title>
+
+  <para>
+   This procedure is intended to be used with public cloud
+   installations. Use SSH to log into the admin node and run the
+   <command>caasp-admin-setup</command> executable as the
+   <literal>root</literal>user.
+  </para>
+
+  <para>
+   By default the <command>caasp-admin-setup</command> executable operates in
+   <literal>wizard</literal> mode, walking you through the necessary steps.
+   During this process your &scc; credentials will be requested. Registration
+   with &scc; can be skipped. If this step is skipped during setup the admin node
+   and the cluster nodes will not receive any updates. While registration to
+   &scc; can be performed after the initial setup with
+   <literal>SUSEConnect</literal>, performing the registration during setup has
+   the advantage that cluster nodes will automatically be registered with &scc;
+   as well. If you prefer not to run the <literal>wizard</literal>, use
+   <command>caasp-admin-setup --help</command> to obtain a list of the
+   available command line arguments.
+  </para>
+
+  <para>
+   Once the <literal>caasp-admin-setup</literal> process is complete all
+   &productname; containers will be launched on the admin node instance. Use
+   your web browser to access the Velum dashboard via <literal>https</literal>.
+   If you did not provide your own certificate, a certificate was generated for
+   you and the fingerprint was written to the terminal in which
+   <command>caasp-admin-setup</command> was executed. You can compare this
+   fingerprint in your browser to establish the chain of trust.
+  </para>
+
+  <para>
+   Microsoft Azure does not provide a time protocol service. Please refer to
```

Comment:
This paragraph needs to be moved into the "General Public Cloud Requirements" subsection for Azure